src/Monomer/Core/Themes/BaseTheme.hs

```diff
@@ -65,6 +65,7 @@ data BaseThemeColors = BaseThemeColors {
   dialogText :: Color,
   dialogTitleText :: Color,
   emptyOverlay :: Color,
+  shadow :: Color,
```

Comment:
Should this type have some prefix on its fields, since they're all exported and may cause conflicts? It's something I've had in my mind for a while, but I've always postponed it.

What do you think?

Reply:
Sounds good! I'll add prefix/lenses in a follow-up PR. Thanks for the recommendation!